### ISSUE TITLE
dashboard/students/ fails when accessing English localization

### ISSUE THREAD

**Issue by exploide** [d120/pyophase#53]
Something weird is going on there. German version of dashboard/students/ works fine, English crashes with
```
NoReverseMatch at /dashboard/students/

'Students' is not a registered namespace inside 'dashboard'
```

@ckleemann @bhaettasch  ???

**Comment by ckleemann** [comment]
Accidentally I marked the app_name variable as translatable string. The commit message should be "Do not translate app_name in students dashboard. fixes #53"

**Comment by exploide** [comment]
And doesn't it mean that makemessages should be run again?

I'm a bit confused. It destroys the Language-Team header (don't know how to separate multiple authors) and it seems that the string you just removed is not deleted but commented out?!

Can you have a look?
